source/gridfs/tests/README.rst

```diff
@@ -1,66 +1,37 @@
+============
 GridFS Tests
 ============
 
-The YAML and JSON files in this directory are platform-independent tests
-meant to exercise a driver's implementation of GridFS.
-
-Format
-======
+.. contents::
 
-Each test file has two top level sections:
+----
 
-1. data
-2. tests
-
-The data section defines the initial contents of the files and chunks
-collections for all tests in that file.
+Introduction
+============
 
-The tests section defines the tests to be run. The format of the tests
-section will vary slightly depending on what tests are being defined.
-In general, they will have the following sections:
+The YAML and JSON files in this directory are platform-independent tests
+meant to exercise a driver's implementation of GridFS. These tests utilize the
+`Unified Test Format <../../unified-test-format/unified-test-format.rst>`__.
 
-1. description
-2. arrange
-3. act
-4. assert
+Conventions for Expressing Binary Data
```

Comment:
This section was written with [this comment from SPEC-1216](https://jira.mongodb.org/browse/SPEC-1216?focusedCommentId=2142974&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-2142974) in mind.